### PR TITLE
Fix issue #12080, !global variable declarations generating warnings with SASSC.

### LINKED ISSUE #12080: Sass !global variable declaration deprecation warning
foundation-sites 6.6.3
 
After a LibSass 3.6.4 update I started receiving deprecation warnings, an example follows:

DEPRECATION WARNING on line 120 of foundation-sites/scss/util/_color.scss:
!global assignments won’t be able to declare new variables in future versions.
Consider adding $secondary-color: null at the top level.

snippet reference https://sass-lang.com/documentation/variables
:warning: Heads up!

Compatibility: Dart Sass since 2.0.0 LibSass✗Ruby Sass✗

Older Sass versions allowed !global to be used for a variable that doesn’t exist yet. This behavior was deprecated to make sure each stylesheet declares the same variables no matter how it’s executed.

The !global flag may only be used to set a variable that has already been declared at the top level of a file. It may not be used to declare a new variable.

### PR DESCRIPTION
**Pull request by ycrepeau**
- Closes https://github.com/foundation/foundation-sites/issues/12080

## Description

The last version of sassc issues a warning each time a !global variable is set, except if such a setting is done at the root level.

The current pull-request add a patches to 3 files by déclaring the variables at root level.

## Types of changes

- [ ] Documentation
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (anything that would change an existing functionality)
- [ ] Maintenance (refactor, code cleaning, development tools...)

## Checklist

- [x] I have read and follow the CONTRIBUTING.md document.
- [x] The pull request title and template are correctly filled.
- [x] The pull request targets the right branch (`develop` or `develop-v...`).
- [x] My commits are correctly titled and contain all relevant information.
- [ ] I have updated the documentation accordingly to my changes (if relevant).
- [ ] I have added tests to cover my changes (if relevant).

### CODE REVIEW

File: package.json
Comment (DanielRuf) [inline package.json:164]: Please revert this change in package.json and also the yarn.lock changes.

Comment (ycrepeau) [reply package.json]: Done.
However, I have failed the CI/ node latest with browserstack (pull request) test. I have a "Error: Configuration parameter username is required." in the log for that test. I don't know what I should do (provide a user name, how??) in this case.

Comment (DanielRuf) [reply package.json]: The failing browserstack test is normal. This happens because forks can not use the secrets in our repo due to security reasons and so the test can not read the credentials from the repo setting on our side.

After the changes are merged a new CI build is done in the repo which runs the browserstack test.

So nothing to worry about.

**Comment by kagarmoe** [comment]
Hi - I'd love to get this patch sometime soon! Thanks.

**Comment by ycrepeau** [comment]
> Hi - I'd love to get this patch sometime soon! Thanks.

So do I.  The patch has been ready since January 24th. 2021. 

Maybe I have missed some details on the approval process and I don't know what I have to do to have is issue finally resolved.

**Review by joeworkman — APPROVED** [review]

**Pull request merged**
